internal/hcsoci/create.go

```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/Microsoft/hcsshim/internal/guid"
 	"github.com/Microsoft/hcsshim/internal/hcs"
 	"github.com/Microsoft/hcsshim/internal/oci"
-	"github.com/Microsoft/hcsshim/internal/schema2"
+	hcsschema "github.com/Microsoft/hcsshim/internal/schema2"
```

Comment:
I don't really care but just wondering why you switches all these? `schema2` made sense to me.

Reply:
I didn't. go fmt did.

Reply:
No worries. Its fine.